docs/guides/modules/orchestrate/pages/pipeline-variables.adoc

```diff
@@ -124,7 +124,7 @@ In addition to using the API, you can also trigger a pipeline with parameters fr
 . Use the *Add Parameters* dropdown to specify the type, name, and value of your desired parameters.
 . Select *Trigger Pipeline*.
 
-Parameters can also be called when setting up a scheduled pipeline in the web app. The parameters are part of the trigger form in menu:Project Settings[Triggers]. Any parameter set up as a part of a scheduled pipeline will also need to be declared in the configuration file, otherwise the pipeline will fail with the error `Unexpected argument(s)`.
+Parameters can also be called when setting up a scheduled pipeline in the web app. The parameters are part of the trigger form in menu:Project Settings[Triggers] or menu:Project Settings[Project Setup]. Any parameter set up as a part of a scheduled pipeline will also need to be declared in the configuration file, otherwise the pipeline will fail with the error `Unexpected argument(s)`.
```

Comment:
```suggestion
Parameters can also be passed when setting up a scheduled pipeline in the web app. The parameters are part of the trigger form in menu:Project Settings[Triggers] or menu:Project Settings[Project Setup]. Any parameter set up as a part of a scheduled pipeline will also need to be declared in the configuration file, otherwise the pipeline will fail with the error `Unexpected argument(s)`.
```